src/api/VoteMonitor.Api.Observer/Controllers/ObserverController.cs

```diff
@@ -157,6 +157,27 @@ public async Task<IActionResult> DeleteObserver(int id)
             return Ok(result);
         }
 
+        /// <summary>
+        /// Removes mobile device Id associated with Observer of given Id.
+        /// </summary>
+        /// <param name="id">The Observer id</param>
+        /// <returns>Boolean indicating whether or not the mobile device Id was removed successfully</returns>
+        [HttpPost]
+        [Route("removeDeviceId")]
+        [Authorize("NgoAdmin")]
+        [Produces(type: typeof(bool))]
+        public async Task<IActionResult> RemoveObserverDeviceId(int id)
+        {
+            if (!ModelState.IsValid)
```

Comment:
This check is not needed as there because of two reasons: there is no validation added to the request (int id) and this is already done by ASP.NET before reaching this action